roles/cinder-data/tasks/main.yml

```diff
@@ -22,12 +22,12 @@
             dest=/etc/cinder/cinder.encryption.conf mode=0640
             owner=root group=cinder
   notify: restart cinder services
-  when: cinder.fixed_key is defined
+  when: cinder.encryption.enabled|bool and cinder.encryption.fixed_key is defined
```

Comment:
I am not pretty sure, but I think we don't need encryption configuration for cinder-data.